### PR TITLE
[10.0][IMP] web_responsive: Load default kanban view for small screen web_responsive.

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@sergio-teruel I'm afraid something is not working properly. Now you can't access to several sections in mobile view. Try the following:

- Put your browser in mobile view.
- Go to main section "Settings".
- Click on "Browse Apps" button. It doesn't work
- Now go to main section "Apps". It also doesn't work.

cc @Yajo

**Comment by sergio-teruel** [comment]
@pedrobaeza ready for testing....

File: web_responsive/README.rst [hunk 1]
```diff
@@ -12,6 +12,8 @@ Features:
 
  * New navigation with an App drawer
  * Keyboard shortcuts for easier navigation
+ * Display kanban views for small screens format if an action or field One2x
```
Comment (lasley) [inline web_responsive/README.rst]: I think remove "format"

> Display kanban views for small screens if an action or field One2x

File: web_responsive/README.rst [hunk 2]
```diff
@@ -52,6 +54,8 @@ this module.
 * Adding ``oe_main_menu_navbar`` ID to the top navigation bar triggers some
   great styles, but also `JavaScript that causes issues on mobile
   <https://github.com/OCA/web/pull/446#issuecomment-254827880>`_
+* The kanban view not is shown if an action window has no kanban mode view
```
Comment (lasley) [inline web_responsive/README.rst]: IMO this isn't worth noting as an issue here. It's inferred that a Kanban won't be shown if there's not a Kanban to show

File: web_responsive/static/src/js/web_responsive.js [hunk 3]
```diff
@@ -286,10 +289,24 @@ odoo.define('web_responsive', function(require) {
         new AppDrawer();
     });
 
+    // if we are in small screen change default view to kanban if exists
+    ViewManager.include({
+        get_default_view: function() {
+            var default_view = this._super()
+            if (config.device.size_class <= config.device.SIZES.XS &&
+                default_view.type != 'kanban' &&
+                this.views['kanban']){
```
Comment (lasley) [inline web_responsive/static/src/js/web_responsive.js]: Totally stylistic, but it just took me a minute to trace these brackets. Please put the `{` on a new line, at same tab level as its closing bracket

**Review by chienandalu — APPROVED** [review]
Tested on runbot on Android Chrome and seems to work fine.

**Comment by sergio-teruel** [comment]
@lasley Changes done, thanks for your review.

**Review by lasley — APPROVED** [review]
Thanks @sergio-teruel

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**